src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -1585,6 +1588,7 @@
     <Field Name="Expanded" Type="bool"/>
     <Field Name="InvokedAsExtensionMethod" Type="bool"/>
     <Field Name="ArgsToParamsOpt" Type="ImmutableArray&lt;int&gt;" Null="allow"/>
+    <Field Name="DefaultArgumentsOpt" Type="BitVector" Null="allow" />
```

Comment:
>Null="allow" [](start = 55, length = 12)

What does `Null="allow"` mean for a value type?

Reply:
On `ImmutableArray<T>` it means that arguments where `IsDefault` returns true will be rejected.

It turns out that on other value types, we default to `NullHandling.NotApplicable`, which means it's not necessary to use this attribute.